### PR TITLE
#171195669 Optimize like and dislike functionality

### PR DESCRIPTION
**Pull request by william000000**
#### What does this PR do?
Change the color according to whether the user like or dislike
#### Description of Task to be completed?
- User can like and dislike and know whether s/he has performed it
#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-frontend.git`
- `cd team-odd-bn-frontend`
- `git checkout ch-improvement-of-like-and-dislike-171195669`
- `npm install` packages
- `npm run dev`
-  Login to the system
- Navigate to the bookings
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#171195669](https://www.pivotaltracker.com/story/show/171195669)
#### Screenshots (if appropriate)
image

#### Questions:
- N/A

### CODE REVIEW

**Review by hezronkimutai — APPROVED** [review]
Nice work @william000000

File: src/assets/css/selectOption.scss [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
 .selectOption {
-  height: 40px !important;
+  height: 30px !important;
   width: 100%;
   border: 1px solid $logo-color;
   outline: none;
   font-size: 14px;
   font-weight: bold;
-}
+}
```
Comment (victkarangwa) [inline src/assets/css/selectOption.scss]: I think you forgot to add EOL

**Review by victkarangwa — COMMENTED** [review]
You did really nice job @william000000 
Just have a look a the feedback I left

**Review by victkarangwa — APPROVED** [review]

File: src/views/accommodation/ViewAccomodation.js [hunk 2]
```diff
@@ -105,9 +105,11 @@ available rooms
                         <img className="dislike_icon" src={disLikeIcon} alt="dislike icon" />
                       </div>
                     </section>
+                    {console.log('what comes---', items)
+                    }
```
Comment (NiyongaboEric) [inline src/views/accommodation/ViewAccomodation.js]: Remove console.

File: src/views/trips/ManagerView.js [hunk 3]
```diff
@@ -56,25 +57,26 @@ const AvailRequestsView = ({
...
         className="search"
         id="searchInput"
         type="text"
         onChange={(e) => {
-          const newData = handleFilter(e, data, search.key);
+          const newData = handleFilter(e, data, search.key || 'manager.trips.any');
           searchResults(newData);
         }}
-        disabled={!search.key && 'disabled'}
+        // disabled={!search.key && 'disabled'}
```
Comment (NiyongaboEric) [inline src/views/trips/ManagerView.js]: Remove comment.

**Review by NiyongaboEric — COMMENTED** [review]
Good job @william000000! You still need only to clean up code.

**Review by NiyongaboEric — APPROVED** [review]

**Pull request merged**